engine/api/worker_model.go

```diff
@@ -146,33 +146,33 @@ func (api *API) bookWorkerModelHandler() service.Handler {
 
 func (api *API) spawnErrorWorkerModelHandler() service.Handler {
 	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
-		spawnErrorForm := &sdk.SpawnErrorForm{}
-		if err := service.UnmarshalBody(r, spawnErrorForm); err != nil {
+		var spawnErrorForm sdk.SpawnErrorForm
+		if err := service.UnmarshalBody(r, &spawnErrorForm); err != nil {
 			return sdk.WrapError(err, "Unable to parse spawn error form")
 		}
 
-		workerModelID, errr := requestVarInt(r, "permModelID")
-		if errr != nil {
-			return sdk.WrapError(errr, "spawnErrorWorkerModelHandler> Invalid permModelID")
+		workerModelID, err := requestVarInt(r, "permModelID")
+		if err != nil {
+			return err
 		}
 
-		tx, errBegin := api.mustDB().Begin()
-		if errBegin != nil {
-			return sdk.WrapError(errBegin, "spawnErrorWorkerModelHandler> Cannot start transaction")
+		tx, err := api.mustDB().Begin()
+		if err != nil {
+			return err
```

Comment:
This should at least be return like sdk.WithStack(err), same for other errors never wrapped.

Reply:
```suggestion
			return sdk.WithStack(err)
```